nym-validator-rewarder/src/rewarder/credential_issuance/monitor.rs

```diff
@@ -93,29 +92,20 @@ impl CredentialIssuanceMonitor {
         // check if this deposit even exists
         let deposit_tx = issued_credential.credential.tx_hash;
 
-        let (deposit_value, deposit_info) = self
+        //not using value anymore, but it should still be there
+        let (_deposit_value, deposit_info) = self
             .nyxd_client
             .get_deposit_transaction_attributes(deposit_tx)
             .await?;
         trace!("deposit exists");
 
         // check if the deposit values match
```

Comment:
my brain is a bit fried at the moment, so can't think of it myself, but do we maybe need to introduce additional, ecash-specific checks here now?

Reply:
I guess this will be linked to the whole economic stuff that will come later. 
For now, we only need to check that there is a deposit, and that it's the correct person using it